state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -652,4 +719,4 @@ def burn(self, burn_num, dutycycle=0, freq=1000, duration=1):
 
 
 # initialize Satellite as pocketqube
-pocketqube = Satellite()
+# pocketqube = Satellite()
```

Comment:
I don't like this change because it could lead us to accidentally creating multiple instances of the cubesat object. And furthermore it makes an unnecessary incompatibility with the pycubed.py library.

Reply:
The point of this change is to give us the ability to initialize the Satellite in different ways (with test mode enabled or without), in the main file. This is assuming that we don't want to have to edit the library. 

The goal was to print more initialization information if testmode was enabled (so we could just use the pycubed.py library file as our hardware test, instead of creating a duplicate version of it to test)

I personally am not the biggest fan of moving the Satellite object initialization to the main file either -- due to the incompatibility issues you mentioned -- but I also think it makes more sense for the cubesat to be initialized in the main file rather than the library.

Reply:
The only other way to do this (and maintain Satellite object initialization within the library) is for us to have different versions of pycubed.py for testing and for flight. We would lose the ability to test in flight by doing this, but I think that's not an immediate issue that we're trying to solve anyways.

Reply:
>  also think it makes more sense for the cubesat to be initialized in the main file rather than the library. Initializing in the main will could give us more freedom with our logging / output / parameter specification in future development cycles.

I'm still not really convinced that it makes sense to initialize the cubesat in the main file. The way I look at it is that it is simply a library that we call in order to interface with the hardware, the details should be abstracted from us as much as possible.

> The only other way to do this (and maintain Satellite object initialization within the library) is for us to have different versions of pycubed.py for testing and for flight.

I personally think this is the correct approach, I don't believe in deploying test code into production (or in our case space). I prefer to have a pretty strong separation between test/flight code.